cypress.json

```diff
@@ -7,5 +7,6 @@
     "colors": {
       "sttmBlue": "rgb(1, 102, 155)"
     }
-  }
-}
\ No newline at end of file
+  },
+  "projectId": "7y3f9z"
```

Comment:
What does this add? does this need to a secret?

Reply:
@plug-n-play

Reply:
I don't think this needs to be a secret 

https://docs.cypress.io/guides/dashboard/projects.html#Project-ID